src/freenas/usr/lib/netdata/python.d/smart_log.chart.py

```diff
@@ -258,6 +258,7 @@ def __init__(self, configuration=None, name=None):
         self.disks = list()
         self.runs = 0
         self.do_force_rescan = False
+        self.update_every = 30 * 60
```

Comment:
Let's add a comment here explaining why we're doing this please 👍 (i.e. "smartd daemon only queries drive temps every 30mins so the files won't be updated but once every 30ish minutes") or something alone those lines 😄

Reply:
@themylogin no it's not.